lib/util/include/hse_util/condvar.h

```diff
@@ -16,6 +16,8 @@
 #ifndef HSE_PLATFORM_CONDVAR_H
 #define HSE_PLATFORM_CONDVAR_H
 
+#include <stdlib.h>
+
```

Comment:
Why is stdlib.h suddenly required by condvar.h?

Reply:
abort()

Reply:
Needs pthread.h too!